test/integration/connect/envoy/docs/windows_prposal.md

```diff
@@ -0,0 +1,95 @@
+What happens in Windows? Host networks are not supported. In Docker networks for Windows each container is a separate unit with its own addresses. This impacts not only the execution of the tests, but all the configurations require an update since in most cases they would point to a localhost that will not have the information because it is in another container.
+
+### Linux network diagram
+
+![*Image of consul in green*](./img/linux-arch.png "*Image of consul in green*")
+
+### Windows network diagram
+
+![*Image of consul in green*](./img/windows-arch-current.png "*Image of consul in green*")
+
+### Diagram of the proposed network
+
+![*Image of consul in green*](./img/windows-arch-proposal.png "*Image of consul in green*")
+
+### Windows docker containers
+
+#### Currently
+
+![*Currently*](./img/windows-container-actual.png "*Currently*")
+
+#### Proposed
+
+![*Proposed*](./img/windows-container-proposal.png "*Proposed*")
+
+## Important items to change
+
+* Unification of environment containers
+* Changes when creating services
+* Changes when starting sidecar
+* Change in the way of creating the Workdir Volume
+
+## Advantage
+
+* More sensitive architecture
+* Fewer images and Containers
+* Shorter response time and test completion
+
+## What the changes consist of
+
+### Unification of environment containers
+
+Only two containers will be mounted: *consul-primary* and *verify-primary*.
+The *consul-primary* container brings together the architecture to be tested (Consult, services, instances...)
+verify-primary is the container where the tests are run from. In the new architecture all these tests would point to *consul-primary*, simplifying their logic.
+
+### Changes when creating services
+
+The functions that create the services would receive a change. Today, every time a service is created, a Fortio container is started that simulates said service. In the proposal, those functions would execute a command in the *consul-primary* container with the listening ports for each service.
+
+Example s1:
+
+```
+fortio server -http-port ":8080" -grpc-port ":8079"
+```
+
+Example s2:
+
+```
+fortio server -http-port ":8181" -grpc-port ":8179"
+```
+
+### Changes when starting sidecar
+
+At the time of starting the sidecar of the services until now, an Envoy containment was mounted. The proposal runs this Envoy inside consul-primary. The function would be modified so that instead of raising the container, it executes commands of the type:
```

Comment:
Currently, when starting the service's sidecar proxy an Envoy container is mounted.